### PR TITLE
Issue 306: Update SCM info for using mvn:release

### PR DESCRIPTION
**Pull request by sijie**
Descriptions of the changes in this PR:

- update the SCM info in the pom files, so that mvn:release is able to create release branch.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,8 +29,30 @@
...
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <scm>
+    <connection>scm:git:https://github.com/apache/bookkeeper.git</connection>
+    <developerConnection>scm:git:https://github.com/apache/bookkeeper.git</developerConnection>
+    <url>https://github.com/apache/bookkeeper</url>
+  </scm>
+  <issueManagement>
+    <!-- TODO: add github issues as well -->
```
Comment (eolivelli) [inline pom.xml]: drop TODO ?

File: pom.xml [hunk 2]
```diff
@@ -29,8 +29,30 @@
...
+    <system>jira</system>
+    <url>https://issues.apache.org/jira/browse/BOOKKEEPER</url>
+  </issueManagement>
+  <ciManagement>
+    <system>Travis</system>
```
Comment (eolivelli) [inline pom.xml]: jenkins ?

File: pom.xml (hunk 1 above)
Comment (sijie) [inline pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (sijie) [inline pom.xml]: done

**Comment by eolivelli** [comment]
Very good

**Pull request closed without merging**